packages/docutils/lib/cli/index.ts

```diff
@@ -8,19 +8,20 @@
 
 import {getLogger} from '../logger';
 
+import {fs} from '@appium/support';
 import _ from 'lodash';
+import {sync as readPkg} from 'read-pkg';
 import {hideBin} from 'yargs/helpers';
 import yargs from 'yargs/yargs';
 import {DEFAULT_LOG_LEVEL, LogLevelMap, NAME_BIN} from '../constants';
 import {DocutilsError} from '../error';
 import {build, init, validate} from './command';
 import {findConfig} from './config';
-import {fs} from '@appium/support';
-import {sync as readPkg} from 'read-pkg';
 
 const pkg = readPkg({cwd: fs.findRoot(__dirname)});
-
 const log = getLogger('cli');
+const IMPLICATIONS_FAILED_REGEX = /implications\s+failed:\n\s*(.+)\s->\s(.+)$/i;
```

Comment:
how stable this regex is going to be?

Reply:
I don't know.  this package needs tests.  but the failure mode here is that the message is output verbatim, which is not ideal, but also not breaking.